src/api/cvc4cpp.cpp

```diff
@@ -630,6 +632,16 @@ CVC4::Kind extToIntKind(Kind k)
   }
   return it->second;
 }
+
+uint32_t minArity(Kind k)
+{
+  return CVC4::ExprManager::minArity(extToIntKind(k));
```

Comment:
Coverity might object to that again, I am not sure (because it thinks that there could be negative values).